packages/test/stochastic-test-utils/src/test/generators.spec.ts

```diff
@@ -381,6 +381,33 @@ describe("generators", () => {
 			["c", 1],
 			["d", 1],
 		],
+		[
```

Comment:
these get used in chi-squared tests that the generator behaves as expected below.